CONTRIBUTING.md

```diff
@@ -2,6 +2,15 @@
 The SPRAS maintainers welcome external contributions.
 Code contributions will be licensed using the project's [MIT license](LICENSE).
 
+If you are looking for the pathway reconstruction guide,
+go to the [Contributing a new pathway reconstruction guide](#contributing-a-new-pathway-reconstruction-guide) header.
+
+## On `.test_durations`
+
+To regenerate the `.test_durations` file, use `pytest --store-durations`. A clean docker cache
```

Comment:
These instructions are great. Please also add a sentence stating what this file does (i.e. used by pytest to decide how to split the tests into k groups, right?).